vagrant/boxes.d/99-local.yaml.example

```
@@ -10,6 +10,20 @@ centos7-katello-devel:
       foreman_devel_github_push_ssh: True
       katello_devel_github_username: <REPLACE ME>
 
+centos8-devel:
+  primary: true
+  box: centos8
+  disk_size: 40
+  ansible:
+    playbook: 'playbooks/katello_devel.yml'
+    group: 'devel'
+    variables:
+      ssh_forward_agent: true
+      foreman_devel_github_push_ssh: True
+      katello_devel_github_username: ehelms
```

Can we use a more generic example?
```suggestion
      katello_devel_github_username: <REPLACE ME>
```